go.mod

```diff
@@ -3,14 +3,14 @@ module github.com/mendersoftware/go-lib-micro
 go 1.18
 
 require (
-	github.com/ant0ine/go-json-rest v3.3.3-0.20170913041208-ebb33769ae01+incompatible
+	github.com/ant0ine/go-json-rest v3.3.2+incompatible
```

Comment:
This is actually downgrading `go-json-rest`. [Comparing the two](https://github.com/ant0ine/go-json-rest/compare/v3.3.2..master), there is one insignificant change fix there so I don't think it really matters.